kml/serdes_binary.hpp

```diff
@@ -251,7 +251,7 @@ class DeserializerKml
     NonOwningReaderSource source(reader);
     m_header.Deserialize(source);
 
-    if (m_header.m_version == Version::V8)
+    if (m_header.m_version == Version::V8 || m_header.m_version == Version::V9)
```

Comment:
Let's add a comment that we likely discover an issue and a need for a separate if later )